Remote Control service not starting

### Current behavior

Remote control service isn't starting because it's looking for the host library in the wrong location
C:\Users\NickRandolph\.nuget\packages\uno.winui.remotecontrol\5.0.0-dev.1249\tools\rc\host\netcoreapp3.1\Uno.UI.RemoteControl.Host.dll


### Expected behavior

Remote control service should start
Valid locations are net7.0 and net8.0 folder
Logic in Entrypoint needs to be updated to (a) pick correct version and (b) check file exists

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

Uno Platform Output

---------------------------
[INFO] Uno Remote Control initialized (5.0.0-dev.1249+Branch.master.Sha.4ce33b81987033de8958de5ddb142ab3e39b8f93.4ce33b81987033de8958de5ddb142ab3e39b8f93)
[INFO] Loaded the Uno.WinUI.RemoteControl Remote Control service (5.0.0-dev.1249).
[ERROR] Could not execute because the specified command or file was not found.
[DEBUG] 
[ERROR] Possible reasons for this include:
[ERROR]   * You misspelled a built-in dotnet command.
[ERROR]   * You intended to execute a .NET program, but dotnet-C:\Users\NickRandolph\.nuget\packages\uno.winui.remotecontrol\5.0.0-dev.1249\tools\rc\host\netcoreapp3.1\Uno.UI.RemoteControl.Host.dll does not exist.
[ERROR]   * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
[ERROR] 
----------------------

--- a/src/Uno.UI.RemoteControl.VS/EntryPoint.cs
+++ b/src/Uno.UI.RemoteControl.VS/EntryPoint.cs
@@ -260,7 +260,8 @@ namespace Uno.UI.RemoteControl.VS
 			{
 				RemoteControlServerPort = GetTcpPort();
 
-				var runtimeVersionPath = GetDotnetMajorVersion() > 5 ? "netcoreapp3.1" : "net6.0";
+				var version = GetDotnetMajorVersion();
+				var runtimeVersionPath = version <= 5 ? "netcoreapp3.1" : $"net{version}.0";
 
 				var sb = new StringBuilder();
 

--- a/src/Uno.UI.RemoteControl.VS/EntryPoint.cs
+++ b/src/Uno.UI.RemoteControl.VS/EntryPoint.cs
@@ -428,7 +428,7 @@ namespace Uno.UI.RemoteControl.VS
 			try
 			{
 				_dte.Events.BuildEvents.OnBuildDone -= _onBuildDoneHandler;
-				_dte.Events.BuildEvents.OnBuildProjConfigBegin += _onBuildProjConfigBeginHandler;
+				_dte.Events.BuildEvents.OnBuildProjConfigBegin -= _onBuildProjConfigBeginHandler;
 			}
 			catch (Exception e)
 			{

